easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur phyx-1.01.orig/src/Makefile.in phyx-1.01/src/Makefile.in
```

Comment:
@omula Please add missing patch description + author or upstream reference.

Also, shouldn't this be `-lopenblas` instead? Do you really need to link to ScaLAPACK?

Reply:
Thanks @boegel 
You are right. It links with openblas I have checked with ldd.